api/envoy/api/v2/listener.proto

```diff
@@ -236,4 +237,8 @@ message Listener {
   // This issue was fixed by `tcp: Avoid TCP syncookie rejected by SO_REUSEPORT socket
   // <https://github.com/torvalds/linux/commit/40a1227ea845a37ab197dd1caffb60b047fa36b1>`_.
   bool reuse_port = 21;
+
+  // Configuration for :ref:`access logs <arch_overview_access_logs>`
+  // emitted by the this listener.
+  repeated config.filter.accesslog.v2.AccessLog access_log = 22;
```

Comment:
Should this be attached to a FilterChain instead of a listener? There is a lot more context about what the traffic is after the FilterChainMatch has been made.

Reply:
I'd like to capture a missing filter chain case.